skgrid/Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM python:3.10
+RUN pip install --upgrade pip
+RUN pip install sklearn pandas pyyaml argparse numpy PyYAML
 
-RUN pip install sklearn pandas
+WORKDIR /skgrid/
 
-ADD create_grid.py run_learn.py runner.py run_train.py /scripts/
\ No newline at end of file
+COPY image_scripts  ./
+COPY data ./data
+COPY predict.sh prediction_runner.py pick_model.py select_model.yml ./
+CMD chmod -R o+r ./
+CMD chmod 777 ./image_scripts/train.sh
+CMD chmod 755 ./select_model.yml
+
+ENTRYPOINT ["/skgrid/image_scripts/train.sh"]
+
+ENTRYPOINT ["/skgrid/predict.sh"]
```

Comment:
I want to update the Dockerfile to only have one entry point. should we split into 2 docker images (one entry point per image) and switch to docker-compose?

Reply:
fixed in merged PR#9